### ISSUE TITLE
[Bug] Landscape in apps keyboard?

### ISSUE THREAD

**Issue by ghost** [peterhajas/MobileNotifier#234]
when you are, lets say, for example, in a landscape game, and you get a message, if you click on the popup, the mobilereply shows up in portrait mode while the keyboard shows up in landscape and hides half of the app, can that be fixed? otherwise it is impossible to send or view a message while in a landscape app!

**Comment by timbob1907** [comment]
I've noticed this too. It would be awesome if you could add support for landscape mode. But in the meantime lock the mobilereply keyboard in portrait somehow.

**Comment by LuisOrozco** [comment]
i came to the sad realization today that i couldnt really use mobilereply in landscape because of this bug. I love landscape mode and its keyboard, so this is a big one for me.

**Comment by btaroli** [comment]
Yes, landscape is a real weakness right now. It would be nice to see it resolved before it leaves beta. The display got so completely confused when I tried doing it that I had to reboot to clear it up. Nasty.
